### PR TITLE
[zh-cn]: create the translation of `element/mtr`

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20746

### CODE REVIEW

File: files/zh-cn/web/mathml/element/mtr/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: <mtr>
+slug: Web/MathML/Element/mtr
+l10n:
+  sourceCommit: 8eece0b998c23e8ea35f936d7371a169974130f5
+---
+
+{{MathMLRef}}
+
+**`<mtr>`** 这个 [MathML](/zh-CN/docs/Web/MathML)元素表示表格或矩阵中的一行。它只能出现在 {{ MathMLElement("mtable") }} 元素中，其子元素是表示单元格的 {{ MathMLElement("mtd") }} 元素。该元素类似于 HTML 中的 {{ HTMLElement("tr") }} 元素。
```
Comment (T34-active) [inline files/zh-cn/web/mathml/element/mtr/index.md:10]:
```suggestion
**`<mtr>`** [MathML](/zh-CN/docs/Web/MathML) 元素表示表格或矩阵中的一行。它只能出现在 {{ MathMLElement("mtable") }} 元素中，其子元素是表示单元格的 {{ MathMLElement("mtd") }} 元素。该元素类似于 HTML 中的 {{ HTMLElement("tr") }} 元素。
```

File: files/zh-cn/web/mathml/element/mtr/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  sourceCommit: 8eece0b998c23e8ea35f936d7371a169974130f5
+---
+
+{{MathMLRef}}
+
+**`<mtr>`** [MathML](/zh-CN/docs/Web/MathML) 元素表示表格或矩阵中的一行。它只能出现在 {{ MathMLElement("mtable") }} 元素中，其子元素是表示单元格的 {{ MathMLElement("mtd") }} 元素。该元素类似于 HTML 中的 {{ HTMLElement("tr") }} 元素。
+
+## 属性
+
+该元素的属性包括 [全局 MathML 属性](/zh-CN/docs/Web/MathML/Global_attributes)。一些浏览器还可能支持以下属性：
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mtr/index.md:14]:
```suggestion
该元素的属性包括[全局 MathML 属性](/zh-CN/docs/Web/MathML/Global_attributes)。一些浏览器还可能支持以下属性：
```

File: files/zh-cn/web/mathml/element/mtr/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- `columnalign` {{Non-standard_Inline}}
+  - : 覆盖此行单元格的水平对齐方式，该水平对齐方式由 {{ MathMLElement("mtable") }} 指定。允许使用以空格分隔的多个值，并应用于相应的列（例如 `columnalign="left center right"`）。可能的值为：`left`、`center` 和 `right`。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mtr/index.md:17]:
```suggestion
  - : 覆盖由 {{ MathMLElement("mtable") }} 对此行单元格指定的水平对齐方式。允许使用以空格分隔的多个值，并应用于相应的列（例如 `columnalign="left center right"`）。可能的值为：`left`、`center` 和 `right`。
```

File: files/zh-cn/web/mathml/element/mtr/index.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- `rowalign` {{Non-standard_Inline}}
+  - : 覆盖此行单元格的垂直对齐方式，该垂直对齐方式由 {{ MathMLElement("mtable") }} 指定。可能的值为：`axis`、`baseline`、`bottom`、`center` 和 `top`。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mtr/index.md:19]:
```suggestion
  - : 覆盖由 {{ MathMLElement("mtable") }} 对此行单元格指定的垂直对齐方式。可能的值为：`axis`、`baseline`、`bottom`、`center` 和 `top`。
```

**Review by yin1999 — COMMENTED** [review]
nits

**Review by yin1999 — APPROVED** [review]

**Pull request merged**